client/src/pages/OurEvents/OurEvents.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "title": "Perkisound Xmas Offer",
+        "img": "events-xmas.jpg",
+        "date" : "23 Dec 2021 - 24 Dec 2021",
+        "text" :"<p>Perkisound is an integrated music and recreation centre.</p><p>WHAT TO EXPECT Come and sing/listen to some live Xmas music in the music room then enjoy the sensory room or outdoor facilities. Find the festive settings on the interactive projector. £5 per child/adult with additional needs. please note this is an event for people with Sen. Not exclusive but very limited numbers per hour; maximum of 4 children or 2 Sen adults per hour. Book via Perkisound messenger or email info@perkisound.org.</p>"
+
```

Comment:
you can use Template literals instead of adding <p> element.

Reply:
Hi @khaledkz  
do you think this will work in case if text should be html formatted with paragraphs and probably decorated with colours etc? im trying to think of an example how to do that, can't come up with the right solution for that.